tests/fork/pool.spec.ts

```diff
@@ -0,0 +1,190 @@
+import { BigNumber } from "ethers";
+import { parseEther } from "ethers/lib/utils";
+import { makeSuite, TestEnv } from "../utils/make-suite";
+import { expect } from "chai";
+import { IERC20, SignerWithAddress, waitForTx } from "../../helpers";
+
+enum RateMode {
+  STABLE = 1,
+  VARIABLE = 2
+}
+
+makeSuite("Pool", (testEnv: TestEnv) => {
+  const zero = BigNumber.from("0");
+  const depositSize = parseEther("10");
+  
+  async function supply(asset: IERC20, user: SignerWithAddress, amount: BigNumber) {
+    const { pool } = testEnv;
+
+    await waitForTx(
+      await asset
+        .connect(user.signer)
+        .approve(pool.address, amount)
```

Comment:
as well you can do an infinite approval to speed up the test and reduce moving parts